less/index.less

```diff
@@ -57,3 +57,172 @@ html, body {
     height: 100%;
 }
 
+
+//Variables
+@color:#D3D3D3;
+@size:30px;
+
+//Mixin
+.position(){
+	font-size:size;
+	padding-bottom: 15px;
+}
+
+.center-column(){
+	display:flex;
+	flex-direction: column;
+	text-align: center;
+}
+
+.container {
+    display:flex;
+	flex-direction: column;
+	align-items: center;
+	width: 70%;
+    margin: 0 auto;    
+    padding-top:30px;
+    padding-bottom:30px;
+    padding-bottom:30px;
+	border-radius:20px;
+	@media (max-width:500px) {
+		.container {  
+		  width: 100%; 
+		 }
+}
+
+header {
+	nav a{
+    padding:30px;
+    text-decoration: none;
+    font-size: 20px;
+    color:black;
+	}
+	@media (max-width:500px) {
+   
+		nav{
+			.center-column;
```

Comment:
Don't forget to use parenthesis here. It _works_ without it, but it is deprecated.